config/seckit.settings.yml

```diff
@@ -0,0 +1,56 @@
+_core:
+  default_config_hash: x6bhN6WZwfVUI_LLMvRJIUW_2c26VTaBozbfXmJWmro
+seckit_xss:
+  csp:
+    checkbox: true
+    vendor-prefix:
+      x: true
```

Comment:
Is that necessary? This results in the addition of deprecated headers like `X-Content-Security-Policy-Report-Only` along the standard ones like `Content-Security-Policy-Report-Only`.

Ref: https://www.tenable.com/plugins/was/112552

Reply:
If I'm reading this right, `X-Content-Security-Policy` is still used by IE, though `X-Webkit-CSP` isn't being used any longer.
https://caniuse.com/contentsecuritypolicy

This is complicated as https://caniuse.com/?search=contentsecuritypolicy-2 doesn't need any vendor prefixes, but the Seckit implementation doesn't use any of the new properties that CSP 2 includes (hashes, nonces, script-src-elem, etc)